dotcom-rendering/src/apps/server/articleToHtml.tsx

```diff
@@ -1,23 +1,41 @@
-import { getScriptsFromManifest } from '../../lib/assets';
+import { isString } from '@guardian/libs';
+import { generateScriptTags, getScriptsFromManifest } from '../../lib/assets';
 import { escapeData } from '../../lib/escapeData';
+import { extractNAV } from '../../model/extract-nav';
 import { makeWindowGuardian } from '../../model/window-guardian';
 import type { FEArticleType } from '../../types/frontend';
+import { ArticlePage } from '../../web/components/ArticlePage';
+import { decideFormat } from '../../web/lib/decideFormat';
 import { renderToStringWithEmotion } from '../../web/lib/emotion';
-import { pageTemplate } from './pageTemplate';
+import { pageTemplate } from '../../web/server/pageTemplate';
 
 export const articleToHtml = (
 	article: FEArticleType,
 ): {
 	clientScripts: string[];
 	html: string;
 } => {
-	const { html, extractedCss } = renderToStringWithEmotion(<></>);
+	const NAV = extractNAV(article.nav);
+
+	const format: ArticleFormat = decideFormat(article.format);
+
+	const { html, extractedCss } = renderToStringWithEmotion(
+		<ArticlePage
+			format={format}
+			article={article}
+			NAV={NAV}
+			renderingTarget="Apps"
+		/>,
+	);
```

Comment:
It is slightly odd to read the apps code and see the 'NAV' property being provided, but not sure that it can be avoided!

Reply:
I've attempted a refactor to omit the prop when `renderingTarget=Apps`. It looks ok!